src/broker/mod.rs

```diff
@@ -89,3 +94,69 @@ pub trait BrokerBuilder {
     /// Construct the `Broker` with the given configuration.
     async fn build(&self) -> Result<Self::Broker, BrokerError>;
 }
+
+// TODO: this function consumes the broker_builder, which results in a not so ergonomic API.
+// Can it be improved?
+/// A utility function to configure the task routes on a broker builder.
+pub(crate) fn configure_task_routes<Bb: BrokerBuilder>(
```

Comment:
We might want to just have a separate `utils` module or something. Not a high priority though.

Reply:
I'll leave it there for now but feel free to move it somewhere else later on